content/en/logs/archives/reloading/s3.md

```diff
@@ -0,0 +1,151 @@
+
+1. You can only reload up to 300 Million log events with one reload job.
+2. You can only reload archived content that is older than 24 hours. 
+
+## Reload Query Rules
+
+Reload query filters can include any of the following arguments:
+
+* `service`
+* `source`
+* `status`
+* `host`
+* any attribute beginning with `@`
+* raw-text searches (which are applied on the `message` attribute)
+
+The following rules currently apply to what reload queries are supported:
+
+1. Reloading by tags is not yet supported.
+2. Use of wildcards is supported.
+3. Special characters (`+ - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /`) [must be properly escaped][7].
+4. Querying by `service` is required.
+
+### Reload Query Examples
+
+For example, the following reload query will reload all log events from the "foo" and "bar" services whose `http.url_details.path` attribute includes the string "my/endpoint" and whose message includes the string "GET":
+
+`service:(foo OR bar) @http.url_details.path:*my\/endpoint* GET`
+
+The following reload queries are not currently supported:
+
+* `@http.url_details.path:*my\/endpoint* GET` <br>(missing `service` argument)
+* `service:(foo OR bar) env:prod @http.url_details.path:*my\/endpoint* GET` <br>(uses a tag, `env`, or else does not properly escape `:` in the raw text search for `env:prod`, <br>should be `env\:prod` or `"env:prod"`)
+* `service:(foo OR bar) GET http://myapp:1234/users` <br>(does not properly escape the `:` and `/` characters in the raw text search for `GET http://myapp:1234/users`, <br>should be `GET http\:\/\/myapp\:1234\/users` or `"GET http://myapp:1234/users"`)
+
+## Historical View Names
+
+The following rules currently apply to how you can name historical views: 
+
+1. names must start with a lower-case letter
+2. names can only contain lower-case alpha-numerical characters or the "-" character
+3. names must be unique (including formerly deleted views)
+
+## Using Historical Views
+
+Once a reload job is completed, you can find and use the reloaded content in the Log Explorer exactly as you can all your other indexed log content. You can select your Historical View from the index selector, and you can use the time range selector to expand to the relevant time-frame. The Logs Search, Patterns, and Analytics features are all fully functional on reloaded content for in-depth investigations. 
+
+## Deleting Historical Views
+
+When you are finished with your reloaded log content and no longer wish to retain it in hot storage in your Datadog account, you can schedule it for deletion, which will occur 24 hours later. To do this, go to [the Historical Views tab][4], hover over a historical view and select and confirm its deletion from the trash can icon on the far right. If you or a teammate wish to unschedule this Historical View from deletion, they have 24 hours to do so by selecting the "undo" icon at the far-right of the historical view. 
```

Comment:
- I think you can drop the reference to "hot storage": `s/and no longer wish to retain it in hot storage in your Datadog account//`
- You shouldn't need to explain the hover action. I think directing the user to click the `trash can` or `delete` button should be enough
- `s/or a teammate//`